src/HttpKernel/RectorKernel.php

```diff
@@ -34,6 +34,12 @@ public function getCacheDir(): string
         return sys_get_temp_dir() . '/_rector';
     }
 
+    public function getLogDir(): string
+    {
+        // manually configured, so it can be replaced in phar
+        return sys_get_temp_dir() . '/_rector';
```

Comment:
Log dir should be different one than cache, to make finding logs easier. By convention

`_rector_log` should do